libnavigation-base/src/test/java/com/mapbox/navigation/base/trip/model/alert/BorderCrossingAdminInfoTest.kt

```diff
@@ -0,0 +1,19 @@
+package com.mapbox.navigation.base.trip.model.alert
+
+import com.mapbox.navigation.testing.BuilderTest
+import org.junit.Test
+
+class BorderCrossingAdminInfoTest :
+    BuilderTest<BorderCrossingAdminInfo, BorderCrossingAdminInfo.Builder>() {
+    override fun getImplementationClass() = BorderCrossingAdminInfo::class
+
+    override fun getFilledUpBuilder() = BorderCrossingAdminInfo.Builder(
+        "code",
+        "codeAlpha3"
```

Comment:
nit: put real values. that way the tests work with a real ISO_3166 lookup table. It also gives readers an idea of the expected values.

```suggestion
        "JP",
        "JPN"
```